packages/juce-blueprint/babel.config.js

```diff
@@ -0,0 +1,20 @@
+module.exports =  {
+  presets: [
+    "@babel/preset-env",
+    "@babel/preset-react",
+    "@babel/preset-typescript",
+  ],
+  plugins: [
+    "@babel/plugin-proposal-class-properties",
+    "@babel/plugin-proposal-object-rest-spread",
```

Comment:
Not sure if the spread plugin is actually required here ... recommended in most setup tutorials.

Reply:
:+1: Happy to have it available